score/ipython/ejscreen_etl.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "20aa3891",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "import requests\n",
+    "import zipfile\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import csv\n",
+    "\n",
+    "data_path = Path.cwd().parent / \"data\"\n",
+    "fips_csv_path = data_path / \"fips_states_2010.csv\"\n",
+    "csv_path = data_path / \"dataset\" / \"ejscreen_2020\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "67a58c24",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "c:\\opt\\justice40-tool\\score\\venv\\lib\\site-packages\\urllib3\\connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'gaftp.epa.gov'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings\n",
+      "  warnings.warn(\n"
+     ]
+    }
+   ],
+   "source": [
+    "download = requests.get(\"https://gaftp.epa.gov/EJSCREEN/2020/EJSCREEN_2020_StatePctile.csv.zip\", verify=False)\n",
+    "file_contents = download.content\n",
+    "zip_file_path = data_path / \"tmp\"\n",
+    "zip_file = open(zip_file_path / \"downloaded.zip\", \"wb\")\n",
+    "zip_file.write(file_contents)\n",
+    "zip_file.close()"
```

Comment:
For folders that are on `.gitignore` I am adding manually an `__init__.py` so that folder is created on checkout. Looks I forgot to force att to `/tmp`. Added now.